src/main/java/org/apache/sling/jcr/repoinit/impl/RepositoryInitializer.java

```diff
@@ -115,6 +118,8 @@ public void processRepository(SlingRepository repo) throws RepositoryException {
                         throw new RepoInitException("Error executing repoinit from " + reference, e);
                     }
                 }
+                Duration duration = Duration.between(start, Instant.now());
```

Comment:
i would move it out of the try-finally block and record timing also in case of failure.
if possible include a hint if it failed or succeeded